modules/docs/arrow-docs/docs/docs/datatypes/statet/README.md

```diff
@@ -136,7 +136,7 @@ import arrow.instances.either.monadError.*
 
 fun stackOperationsS2() = 
  ForStateT<EitherPartialOf<StackError>, Stack, StackError>(Either.monadError<StackError>()) extensions {
-  binding {
+  binding<StateTPartialOf<EitherPartialOf<StackError>, Stack>, String> {
```

Comment:
Something broke here with inference. The compiler trips here without redundantly specifying the types, this is unrelated to Ank. Also IntelliJ tells you this is redundant...